packages/common/src/extensions/__tests__/slickCellRangeSelector.spec.ts

```diff
@@ -701,4 +705,52 @@ describe('CellRangeSelector Plugin', () => {
       done();
     }, 7);
   });
+
+  it.only('should maintain propagation only if editor is on current cell', () => {
```

Comment:
oops looks like you forgot to remove `.only` after you're done 😉

Reply:
yeah thats super unfortunate ... I've added eslint-plugin-jest to detect those. Than started to fix the main ones by adjusting the the eslintignore file. As soon as I touched __tests__ though, ~250 errors came up. So those should be handled in a separate shot not to mess up this PR

Reply:
hmm yeah that's just linting cosmetic though, it should indeed be better to add ESLint stuff as a separate PR